CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+- On iOS `NSString` was renamed to `NSStringRust` to support the "Debug View Heirarchy" feature of Xcode.
```

Comment:
```suggestion
- On iOS, fixed support for the "Debug View Heirarchy" feature in Xcode.
```

I think it's better to have the CHANGELOG only mention the user-facing aspect of a change, especially in this case, since it could be misinterpreted as an API change.